runtime-config.md

```diff
@@ -15,25 +15,25 @@ Additional filesystems can be declared as "mounts", specified in the *mounts* ar
         "type": "proc",
         "source": "proc",
         "destination": "/proc",
-        "options": ""
+        "options": {}
     },
     {
         "type": "tmpfs",
         "source": "tmpfs",
         "destination": "/dev",
-        "options": "nosuid,strictatime,mode=755,size=65536k"
+        "options": {"nosuid":true,"strictatime":true,"mode":"755","size":"65536k"}
     },
     {
         "type": "devpts",
         "source": "devpts",
         "destination": "/dev/pts",
-        "options": "nosuid,noexec,newinstance,ptmxmode=0666,mode=0620,gid=5"
+        "options": {"nosuid":true,"noexec":true,"newinstance":true,"ptmxmode":"0666","mode":"0620","gid":5}
     },
     {
         "type": "bind",
         "source": "/volumes/testing",
         "destination": "/data",
-        "options": "rbind,rw"
+        "options": {"rbind":true,"rw":true}
```

Comment:
fair, but I think this is more acceptable though as the `mount -o ...` is required as a comma delimited list. So this would put the formating as the greatest common denominator.